frigate/app.py

```diff
@@ -316,7 +316,11 @@ def receiveSignal(signalNumber, frame):
         server = pywsgi.WSGIServer(
             ("127.0.0.1", 5001), self.flask_app, handler_class=WebSocketHandler
         )
-        server.serve_forever()
+
+        try:
+            server.serve_forever()
+        except KeyboardInterrupt:
```

Comment:
This doesn't necessarily belong as a part of this PR, but in testing I noticed some exceptions being raised on `ctrl+C` so this handles it.